diff -up beesu-2.7/nautilus-beesu-manager-1.8/libexec/scripts/Edit with gEdit.scripts-fixes beesu-2.7/nautilus-beesu-manager-1.8/libexec/scripts/Edit with gEdit
--- beesu-2.7/nautilus-beesu-manager-1.8/libexec/scripts/Edit with gEdit.scripts-fixes	2010-07-12 07:06:09.000000000 -0600
+++ beesu-2.7/nautilus-beesu-manager-1.8/libexec/scripts/Edit with gEdit	2013-09-02 23:17:15.060104744 -0600
@@ -4,7 +4,9 @@
 then
     quoted=$(echo -e "$NAUTILUS_SCRIPT_SELECTED_FILE_PATHS" | 
              awk 'BEGIN { FS = "\n" } { printf "\"%s\" ", $1 }' | sed -e s#\"\"##)
-    beesu - gedit "$quoted"
+    beesu - 'unset XDG_RUNTIME_DIR ; gedit' "$quoted"
 fi
 # F12
 /bin/true
+
+
diff -up beesu-2.7/nautilus-beesu-manager-1.8/libexec/scripts/GNOME Shell Open.scripts-fixes beesu-2.7/nautilus-beesu-manager-1.8/libexec/scripts/GNOME Shell Open
--- beesu-2.7/nautilus-beesu-manager-1.8/libexec/scripts/GNOME Shell Open.scripts-fixes	2013-07-28 10:50:08.000000000 -0600
+++ beesu-2.7/nautilus-beesu-manager-1.8/libexec/scripts/GNOME Shell Open	2013-09-17 03:43:07.711492958 -0600
@@ -3,7 +3,7 @@
 then
     quoted=$(echo -e "$NAUTILUS_SCRIPT_SELECTED_FILE_PATHS" | 
              awk 'BEGIN { FS = "\n" } { printf "\"%s\" ", $1 }' | sed -e s#\"\"##)
-    beesu - xdg-open "$quoted"
+    beesu - 'unset XDG_RUNTIME_DIR ; xdg-open' "$quoted"
 fi
 # F12
 /bin/true
diff -up beesu-2.7/nautilus-beesu-manager-1.8/libexec/scripts/Open Nautilus Here.scripts-fixes beesu-2.7/nautilus-beesu-manager-1.8/libexec/scripts/Open Nautilus Here
--- beesu-2.7/nautilus-beesu-manager-1.8/libexec/scripts/Open Nautilus Here.scripts-fixes	2013-07-28 10:50:59.000000000 -0600
+++ beesu-2.7/nautilus-beesu-manager-1.8/libexec/scripts/Open Nautilus Here	2013-09-15 03:10:07.170694082 -0600
@@ -6,6 +6,6 @@
      while [ ! -z "$1" -a ! -d "$base/$1" ]; do shift; done
      dir="$base/$1"
 fi
-beesu - nautilus "'$dir'"
+beesu - 'unset XDG_RUNTIME_DIR ; nautilus' "'$dir'"
 # F12
 /bin/true
